prerequisites.sh

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env bash
+
+CHECK=true
+echo ""
+echo "Welcome! Before you can run our tool, you need to make sure you have all the packages and libraries installed !"
+
+echo ""
+#checks if caliper is installed under /bin, if not it will be installed and also runs the caliper bind to conigure ethereum globally
+
+echo ""
+echo "We will procees further"
+echo ""
+  if hash "jq" 2>/dev/null; then
+    echo 'jq is installed'
+  else
+   CHECK=false;
+    echo 'jq is missing.'
+    echo 'Please follow this link: https://stedolan.github.io/jq/download/  to install jq'
+   
+  fi
+  
+  #Install and initialize the Cloud SDK using this link https://cloud.google.com/sdk/docs/?hl=de and install docker with this link https://docs.docker.com/install/linux/docker-ce/ubuntu/
+
+ if hash "gcloud" 2>/dev/null; then
+    echo 'Google cloud SDK is installed. Please wait a few seconds..' 
+    echo 'Follow the link to log in please!'
+   gcloud auth login
+ echo " We will process.."
+
+  else
+   CHECK=false;
+    echo 'Google cloud SDK is not installed. Please follow the instructions here: https://cloud.google.com/sdk/install'
+  fi
+  
+  
+  if hash "npm" 2>/dev/null; then
+    echo 'npm is installed'
+  else
+   CHECK=false;
+    echo 'npm is missing. Please download Node.js and follow the instructions in the link: https://www.npmjs.com/get-npm '
+  fi
+  
+
+
+    if hash "geth" 2>/dev/null; then
+    echo 'geth is installed'
+  else
+   CHECK=false;
+    echo 'geth is missing. Please follow this link https://github.com/ethereum/go-ethereum/wiki/Installing-Geth to install geth'
+  fi
+  
+    if hash "puppeth" 2>/dev/null; then
+    echo 'puppeth is installed'
+  else
+   CHECK=false;
+    echo 'puppeth is missing. Please follow this link https://www.sitepoint.com/puppeth-introduction/ to install puppeth'
+  fi
+  
+  echo ""
+   cd cp_ws_1920/bin
```

Comment:
error here! wrong path

Reply:
why wrong path?

Reply:
and what comment did i left out? :/

Reply:
@boxrhcp wrong path?

Reply:
I get the following errors.
![image](https://user-images.githubusercontent.com/12150712/72884907-36b33100-3d07-11ea-89e2-b2de01d66dc9.png)

bin is under cp_ws but im running the script inside cp_ws folder.

Reply:
<img width="960" alt="Screen Shot 2020-01-22 at 19 50 47" src="https://user-images.githubusercontent.com/56897059/72924128-adc0e780-3d50-11ea-8af6-abeff49c96af.png">
 corrected that one thing

Reply:
@boxrhcp gracias